src/lang/en-US.ts

```diff
@@ -1156,6 +1156,9 @@ const Details: ResolvedIntlConfig['messages'] = {
     'details.settings.notifications.dataProcessing.noDataProcessedInInterval.intervalOptions.hour': `{interval} {interval, plural, one {Hour} other {Hours}}`,
     'details.settings.notifications.dataProcessing.noDataProcessedInInterval.intervalOptions.day': `{interval} {interval, plural, one {Day} other {Days}}`,
     'details.settings.notifications.dataProcessing.noDataProcessedInInterval.unsetOption': `None`,
+
+    'details.spec.cta.formatted': `Table`,
+    'details.spec.cta.raw': `Code`,
```

Comment:
I contemplated appending both toggle button labels with the word _View_ (i.e., _Table View_ and _Code View_) but that felt a little redundant. Open to any suggestions here.

Reply:
I think the keys are better names honestly. `Formatted` and `Raw` make more sense... feel much more strong about `Code` -> `Raw`.

Reply:
Those were the labels I started with but I was not sure if _raw_ was too strong of a word.

Reply:
Dave signed off on _table_ and _code_ for the time being.

> our docs do not really have a solid way of expressing this. The only other word I see a lot before a block of spec is `file` but that also _kinda_ feels weird.

I noticed this too and believe it is worth keeping an eye on how our terminology evolves.